### PR TITLE
Edge RN. Added Edge RN for 10.13 release.

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

File: content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Download the Cumulocity IoT Edge version 10.13 from [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
+
+### Changing the hostname
+
+You can now change the hostname of your Edge appliance using the user interface. For more information, see [Configuring the hostname using the UI](https://cumulocity.com/guides/edge/configuration/#configuring-the-hostname).
+
+### Configuring the time synchronization
+
+You can now configure the time synchronization of your Edge appliance using the user interface. For more information, [Configuring the time synchronization using the UI](https://cumulocity.com/guides/edge/configuration/#configuring-the-time-synchronization-using-the-ui).
```
Comment (BeateRixen) [inline content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md:17]: For more information, > For more information, see

File: content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### Known issues
+
+|<div style="width:100px">Ticket</div>|<div style="width:200px">Area</div>|Description
+|:---|:---|:---
+|CIE-2962|SSL certificate|**Issue:** Changing the SSL certificate fails. This issue occurs because the ordering of the certificates chain is incorrect.<br><br>**Workaround:** Fix the certificate ordering as described in the [Prerequisites](https://cumulocity.com/guides/edge/installation/#prerequisites) section or generate a self signed certificate. Do not reboot the system before a new certificate has been successfully installed.
+||Installation service|**Issue:** The installation service might return an HTTPS status code 408 error when polling the task in the version 10.11. This is mostly observed during the upgrade.<br><br>**Workaround:** You may ignore the error or poll the `/edge/tasks/latest/update` endpoint until the task is successful.
```
Comment (BeateRixen) [inline content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md:28]: `/edge/tasks/latest/update` > <kbd>/edge/tasks/latest/update<kbd>
We use kbd for endpoints throughout the documentation

File: content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+### Known issues
+
+|<div style="width:100px">Ticket</div>|<div style="width:200px">Area</div>|Description
+|:---|:---|:---
+|CIE-2962|SSL certificate|**Issue:** Changing the SSL certificate fails. This issue occurs because the ordering of the certificates chain is incorrect.<br><br>**Workaround:** Fix the certificate ordering as described in the [Prerequisites](https://cumulocity.com/guides/edge/installation/#prerequisites) section or generate a self signed certificate. Do not reboot the system before a new certificate has been successfully installed.
+||Installation service|**Issue:** The installation service might return an HTTPS status code 408 error when polling the task in the version 10.11. This is mostly observed during the upgrade.<br><br>**Workaround:** You may ignore the error or poll the `/edge/tasks/latest/update` endpoint until the task is successful. 
+
+For new enhancements in Cumulocity IoT, see [What's new](/release-10-11-0/whatsnew-10-11-0/).
```
Comment (BeateRixen) [inline content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md:30]: Should probably point to 10.13?

File: content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md (hunk 1 above)
Comment (sharath-jk) [reply content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md:17]: Done.

File: content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md (hunk 2 above)
Comment (sharath-jk) [reply content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md:28]: Done

File: content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md (hunk 3 above)
Comment (sharath-jk) [reply content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md:30]: Changed.

**Review by MWindel — APPROVED** [review]

**Pull request merged**